api.go

```diff
@@ -20,11 +20,12 @@ import (
 )
 
 type Client struct {
-	Token         string
-	SIOEndpoint   url.URL
-	Http          http.Client
-	Insecure      string
-	ShowBody      bool
+	Token       string
+	SIOEndpoint url.URL
+	Http        http.Client
+	Insecure    string
+	ShowBody    bool
+
 	configConnect *ConfigConnect
 }
 
```

Comment:
Hi @koensayr,

The issue is that the original author (*cough* @clintonskitson *cough*) didn't use `gofmt` to format the sources, and now he does and we all do, and thus the change you see above. This should happen far less frequently as all our code uses `gofmt` by default per the build process not to mention Atom has it built in.